docs/Development-and-Release.md

```diff
@@ -0,0 +1,32 @@
+## Development and Release
+
+The development, versioning and release of Open-RMF follows that of the [ROS 2 project](https://docs.ros.org/en/rolling/).
+
+### Development and versioning
+* The primary development branch is `main` on all repositories. This is the equivalent to `rolling` in ROS 2. Any `rolling` branches in Open-RMF repos should be ignored as they exist purely for legacy reasons.
+* Repositories also have ROS 2 distro branches, eg., `humble`, containing versions of the packages distributed as a binary for that distro.
+* All new features and bug fixes must target `main`. The changes may be backported  via cherry-picks to one or more distro branches provided there are no API/ABI breaks.
+* The version of packages on `main` must always be higher than on distro branches (atleast 1 minor version higher) and newer distro versions must be higher than older ones (again, 1 minor version higher).
+* Only patch version bumps are allowed on distro branches. This is to ensure distro branch versions do not increase higher than newer distros or `main`.
+* Do not update any changelogs.
+
+### Release
+* Open-RMF binaries for new ROS 2 distributions will be available along with the new release of ROS 2.
+* Roughly monthly, patch releases for packages with changes will be available in source and binary forms. All releases will be announced [here](https://github.com/open-rmf/rmf/releases)
+  * Source releases are in the form of a [tagged rmf.repos](https://github.com/open-rmf/rmf/blob/release-humble-230606/rmf.repos) file. The least `rmf.repos` file in `<distro>-release` branches contain the latest tags for all the repos for that `<distro>`. The `release-<distro>-YYMMDD` tags are useful for checking out packages to particular release versions.
+  * Binary packages may be updated by running `sudo apt update && sudo apt upgrade -y`.
+
+
+### Additional information for maintainers
+* We release into ROS 2 Rolling from the `main` source branch and from `<distro>` branches into supported ROS 2 distributions.
+* Reference this [dashbaord](https://osrf.github.io/osr_dashboard/?distribution=rmf-rolling) to get an overview of Open-RMF repositories with any new commits since the last release.
```

Comment:
```suggestion
* Reference this [dashboard](https://osrf.github.io/osr_dashboard/?distribution=rmf-rolling) to get an overview of Open-RMF repositories with any new commits since the last release.
```